validation/datehelpers.go

```diff
@@ -0,0 +1,80 @@
+package validation
+
+import (
+	"regexp"
+	"strconv"
+	"time"
+
+	errorhandler "github.com/cloudtrust/common-service/errors"
+)
+
+type longDuration struct {
+	years  int
+	months int
+	days   int
+}
+
+const (
+	// RegExpLongDuration is a regular expression used to parse long durations
+	//RegExpLongDuration = `^((([1-9]\d*)([ymwd]))+)$`
+	RegExpLongDuration = `([1-9]\d*)([ymwd])`
+)
+
+var (
+	regExpLongDuration = regexp.MustCompile(RegExpLongDuration)
+)
+
+func parseLongDuration(value string) (longDuration, error) {
```

Comment:
Why do we call it longDuration as we are in fact working with a string ?

Reply:
Renamed to LargeDuration + added a comment to better explain what is a LargeDuration (difference with time.Duration)